src/pudl_archiver/archivers/epacems.py

```diff
@@ -86,5 +89,56 @@ async def get_year_quarter_resource(
             f"File no. {request_count}: Downloaded {year} Q{quarter} EPA CEMS hourly emissions data."
         )
         return ResourceInfo(
-            local_path=archive_path, partitions={"year_quarter": f"{year}q{quarter}"}
+            local_path=archive_path,
+            partitions={"year": year, "quarter": quarter},
+            layout=ZipLayout(filepaths=[filename]),
```

Comment:
Blocking: This needs to be `file_paths`, otherwise it produces the following error when running an archive:

```
file_paths
  Field required [type=missing, input_value={'filepaths': ['epacems-1996-2.csv']}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
```

When amended to `file_paths` we get: `No validations defined for files of type: .csv - epacems-1996-4.csv`
Perhaps this is the wrong filename that is being passed, and the method wants the zipfile name rather than the CSV? Check with @zschira that this is being implemented as intended.

Reply:
@zschira I think this line was your addition, any thoughts here?

Reply:
I just checked, and right now we don't actually have a specific validation for CSV's defined, so it's still checking that the file exists, and that it's not empty, but that's all it does. We could add a function that opens up the CSV and reads it in to see that it's truly a valid CSV, but the default checks might be sufficient.